rc/base/markdown.kak

```diff
@@ -113,8 +113,9 @@ addhl -group /markdown/content regex ^(#+)(\h+)([^\n]+) 1:header
 addhl -group /markdown/content regex ^\h?+((?:[\s\t]+)?[-\*])\h+[^\n]*(\n\h+[^-\*]\S+[^\n]*\n)*$ 0:list 1:bullet
 addhl -group /markdown/content regex ^([-=~]+)\n[^\n\h].*?\n\1$ 0:block
 addhl -group /markdown/content regex \B\+[^\n]+?\+\B 0:mono
+addhl -group /markdown/content regex \B\*[^\n]+?\*\B 0:italic
 addhl -group /markdown/content regex \b_[^\n]+?_\b 0:italic
-addhl -group /markdown/content regex \B\*[^\n]+?\*\B 0:bold
+addhl -group /markdown/content regex \B[\*_]{2}[^\n]+?[\*_]{2}\B 0:bold
```

Comment:
This would also match `_*bold sentence**`, you have to split it in 2 regexes, one for `__` and one for `**`.